src/unix/fs.c

```diff
@@ -926,6 +974,9 @@ static ssize_t uv__fs_sendfile(uv_fs_t* req) {
           /* ENOSYS - it will never work */
           errno = 0;
           copy_file_range_support = 0;
+        } else if (r == -1 && errno == EACCES && uv__is_buggy_cephfs(in_fd)) {
+          errno = 0;
+          copy_file_range_support = 0;
```

Comment:
As a small aside: these should use `uv__store_relaxed()` rather than assign directly. I'll do that in a follow-up PR.

Reply:
I opened a PR to use those functions in https://github.com/libuv/libuv/pull/3124. Will update it again once this gets landed. :)